org.eclipse.ui.editors/src/org/eclipse/ui/internal/editors/text/TextEditorDefaultsPreferencePage.java

```diff
@@ -852,6 +854,11 @@ public void widgetSelected(SelectionEvent e) {
 		IntegerDomain tabWidthDomain= new IntegerDomain(1, 16);
 		addTextField(appearanceComposite, tabWidth, tabWidthDomain, 15, 0);
 
+		label= TextEditorMessages.TextEditorPreferencePage_lineSpace;
+		Preference lineSpace= new Preference(AbstractDecoratedTextEditorPreferenceConstants.EDITOR_LINE_SPACE, label, null);
+		IntegerDomain lineSpaceDomain= new IntegerDomain(0, 1000);
```

Comment:
the number can go higher than 100, 200% for "triple space".
The other way to interpret the value could be  : "200%" as" double space", 90%  as "shrink line space by 10%", but since styledtext ignores  negative value on line space ,  input value 0-99 will be ignored and might create more confusion

Reply:
OK, got it.